helm/nats-account-server/templates/nats-account-server.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: v1
+kind: Pod
```

Comment:
Shouldn't this be a Deployment?

Reply:
could be yeah ... i home brewed this helm by reverse engineering the pod def post deployment.

will give it a go.